src/IO/LzmaReadBuffer.cpp

```diff
@@ -0,0 +1,81 @@
+#include <IO/LzmaReadBuffer.h>
```

Comment:
Just a small thing... in our style we keep capital letters in abbreviations, eg.
LZMA, not Lzma
JSON, not Json
HTTP, not Http.